### ISSUE TITLE
Better example usage details

### ISSUE THREAD

**Issue by mnearents** [msukmanowsky/koa-session-firestore#4]
I have a few questions about how to use this as I am not a shopify app development expert:

1. What URL do I point my shopify app to? Right now I'm pointing it to my firebase hosting URL where I've built out the UI of my app, example: https://appname-ya2y2.web.app

2. Where do I put my cloud functions? Do they have to go in server.js? Or can I keep them in index.ts?

3. What is the "super secret key"? I assume that's the secret key for the shopify app I created in my partner dashboard?

4. Can you give an example of the credentials.json file and what goes in there with fake values?

**Comment by mnearents** [comment]
Thinking the answer to 4 is the file you download from firebase console.

**Comment by msukmanowsky** [comment]
Hey @mnearents, I'd recommend reviewing Shopify's [tutorial](https://shopify.dev/tutorials/build-a-shopify-app-with-node-and-react) first. I uses next.js, but you'll hopefully see some of what prompted this library to enable similar functionality for Firebase.

**Comment by mnearents** [comment]
I think I get it now. This isn't a way to create an app using Firebase as the backend (i.e. cloud functions, hosting, etc.). This is a way to talk to Firestore from a node server running somewhere else correct?

**Comment by msukmanowsky** [comment]
Correct! Koa is just a server framework for Node and this gives you a session store backed by Firestore.
